mk/spksrc.spk.mk

```diff
@@ -269,7 +269,7 @@ $(DSM_LICENSE_FILE): $(LICENSE_FILE)
 $(WORK_DIR)/package.tgz: icon service
 	$(create_target_dir)
 	@[ -f $@ ] && rm $@ || true
-	(cd $(STAGING_DIR) && tar cpzf $@ --owner=root --group=root *)
+	(cd $(STAGING_DIR) && find . -mindepth 1 -maxdepth 1 -not -empty | tar cpzf $@ --owner=root --group=root --files-from=/dev/stdin)
```

Comment:
@th0ma7 
We must include empty directories and remove `-not -empty` in this line.
There are several packages that create (empty) directories in the staging area to have those created on package installation.
Identified so far:
- tt-rss creates var/log
- gateone creates ssl
- lirc creates var/log
- ntopng creates var/data
- haproxy creates var/crt
- monit creates var/events
...

Reply:
@th0ma7 you're right, subfolders under var are not a problem. But as I was building gateone packages, those where missing the empty ssl folder (target/ssl).

Reply:
@hgy59 so indeed this would affect the ssl top-directory as being empty for `gateone`... Unless you convert that into `service-setup.sh` directory creation shell call instead?

Reply:
is there any need to exclude empty directories?

Reply:
There was one in one of my previous (and more complex) PR where I was dealing with all other paths as well.  But with the current setup no, I don't believe so.  It most probably can be removed with no harm.

Reply:
As it is not a good solution to put certificates in target/ssl, I will update gateone to use target/var/ssl folder.